portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,33 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <div class="header">
+      <h1>Anthony Zhu</h1>
+
+      <a id="img-link" href="https://www.linkedin.com/in/anthony-zhu-738215175/" target="_blank">
+        <img id="img-link" src="/images/LI-In-Bug.png" alt="LinkedIn Profile" style="width:32px;height:32px;border:0;">
+      </a>
+
+      <a id="img-link" href="https://github.com/anthonyqzhu" target="_blank">
+        <img id="img-link" src="/images/GitHub-Mark-32px.png" alt="GitHub Profile" style="width:32px;height:32px;border:0;">
+      </a>
+      </div>
+      <p> Hello! My name is Anthony and I am a rising junior at the University
+          of Michigan. I am studying Computer Science with minors in Math and
+          Philosophy. In my free time I like to play video games, watch movies,
+          play sports, and eat good food. Welcome to my portfolio, where I will
+          show off some of my accomplishments and interests.
+      </p>
       <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <button onclick="addRandomQuote()">Click me!</button>
+      <div id="quote-container"></div>
+    </div>
+
+    <div class="sidebar">
+        <a href="index.html">Home</a>
+        <a href="interests.html">Interests</a>
+        <a href="#">Photos</a>
+        <a href="#">Education</a>
```

Comment:
nit: Remove these for now and add them back later once you actually write the html files.

Reply:
done